cross-chain/arbitrum/external/goerliArbitrum/TokenBridge.json

```diff
@@ -0,0 +1,74 @@
+{
+  "address": "0x0b2402144Bb366A632D14B83F244D2e0e21bD39c",
```

Comment:
Double-checked with https://book.wormhole.com/reference/contracts.html#token-bridge-1 but looks the address here is wrong and it should be `0x23908A62110e21C04F3A4e011d24F901F911744A`. Is that correct?

Reply:
Good catch [e4ffa9b](https://github.com/keep-network/tbtc-v2/pull/554/commits/e4ffa9b6c69c782a4694a499ca6300c44428b328)